packages/react-notifications/src/components/notifications-list/notifications-list.tsx

```diff
@@ -30,199 +30,209 @@ type Props = {
   domain: TAppNotificationDomain;
 };
 
-const NotificationsList = (props: Props) => {
+const NotificationsListGlobal = (props: Props) => {
```

Comment:
Here I just split up the switch-case into separate components, do avoid rendering conditional hooks